// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`.toHaveBeenCalledWithArgumentsMatching reports a helpful message if a .not test fails with few calls 1`] = `
"expect(received).not.toHaveBeenCalledWithArgumentsMatching(expected)
Expected no call with arguments matching:
      [Any<String>]
Received (1 total):
*000: [\\"arg1\\"]"
`;

exports[`.toHaveBeenCalledWithArgumentsMatching reports a helpful message if a .not test fails with many calls "for a call in the middle" 1`] = `
"expect(received).not.toHaveBeenCalledWithArgumentsMatching(expected)
Expected no call with arguments matching:
      [\\"arg3\\"]
Received (7 total):
 002: [\\"arg2\\"]
*003: [\\"arg3\\"]
 004: [\\"arg4\\"]"
`;

exports[`.toHaveBeenCalledWithArgumentsMatching reports a helpful message if a .not test fails with many calls "for the first call" 1`] = `
"expect(received).not.toHaveBeenCalledWithArgumentsMatching(expected)
Expected no call with arguments matching:
      [\\"arg0\\"]
Received (7 total):
*000: [\\"arg0\\"]
 001: [\\"arg1\\"]
 002: [\\"arg2\\"]"
`;

exports[`.toHaveBeenCalledWithArgumentsMatching reports a helpful message if a .not test fails with many calls "for the last call" 1`] = `
"expect(received).not.toHaveBeenCalledWithArgumentsMatching(expected)
Expected no call with arguments matching:
      [\\"arg6\\"]
Received (7 total):
 004: [\\"arg4\\"]
 005: [\\"arg5\\"]
*006: [\\"arg6\\"]"
`;

exports[`.toHaveBeenCalledWithArgumentsMatching reports a helpful message if a .not test fails with many calls "for the second call" 1`] = `
"expect(received).not.toHaveBeenCalledWithArgumentsMatching(expected)
Expected no call with arguments matching:
      [StringMatching /1/]
Received (7 total):
 000: [\\"arg0\\"]
*001: [\\"arg1\\"]
 002: [\\"arg2\\"]"
`;

exports[`.toHaveBeenCalledWithArgumentsMatching reports a helpful message if a .not test fails with many calls "for the second-to-last-call" 1`] = `
"expect(received).not.toHaveBeenCalledWithArgumentsMatching(expected)
Expected no call with arguments matching:
      [\\"arg5\\"]
Received (7 total):
 004: [\\"arg4\\"]
*005: [\\"arg5\\"]
 006: [\\"arg6\\"]"
`;

exports[`.toHaveBeenCalledWithArgumentsMatching reports a helpful message if a call is not matched 1`] = `
"expect(received).toHaveBeenCalledWithArgumentsMatching(expected)
Expected call with arguments matching:
      [\\"call7\\"]
Received (6 total):
 003: [\\"call4\\"]
 004: [\\"call5\\"]
 005: [\\"call6\\"]"
`;

exports[`.toHaveBeenCalledWithArgumentsMatching reports a helpful message if function was not called 1`] = `
"expect(received).toHaveBeenCalledWithArgumentsMatching(expected)
Expected call with arguments matching:
      [\\"call7\\"]
Received (0 total):"
`;

exports[`.toHaveBeenCalledWithArgumentsMatching throws if argument is "an object" 1`] = `"Test error: toHaveBeenCalledWithArgumentsMatching called with other than jest.Mock"`;

exports[`.toHaveBeenCalledWithArgumentsMatching throws if argument is "not a mock function" 1`] = `"Test error: toHaveBeenCalledWithArgumentsMatching called with other than jest.Mock"`;

exports[`.toHaveBeenCalledWithArgumentsMatching throws if argument is "null" 1`] = `"Test error: toHaveBeenCalledWithArgumentsMatching called with other than jest.Mock"`;
